### DISCUSSION TITLE
POTA DB Update

### DISCUSSION THREAD

**Discussion by JCinOK** [magicbug/Cloudlog#3142] [Ideas / Feature Requests]
All the POTA DB entities were recently updated.  Are there plans to update the POTA references in CloudLog anytime soon to match the new POTA updates?

I realize this might be a bigger problem for users with many POTA references already in their logs.

**Comment by magicbug** [comment]
We get the current list from https://pota.app/all_parks.csv so I'd hope they keep this updated? as per Cloudlogs install guide its recommended that a cron runs every so often this URL https://<URL-To-Cloudlog>/index.php/update/update_pota that downloads the new csv.

But yes it does present a problem as I don't know if a remap list has been provided the whole change seems to have been badly handled by pota themselves.
